lumi/get_slot.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -ex
+
+echo "Requesting slot on LUMI HPC..."
+JENKINS_VERSION=$1
+echo "Jenkins version is $JENKINS_VERSION"
+
+OS=$2
+SCRIPT=$3
```

Comment:
do we need to pass the script name here? isn't it going to be `$(dirname $0)/jenkins_java.sh` always?

Reply:
Yes, you are is right, thanks!